### ISSUE TITLE
css_selector option seems to not work

### ISSUE THREAD

**Issue by wburningham** [DannyBen/snapcrawl#48]
Let me know if I am just using the option incorrectly, but if not it seems like the `css_selector` option is not working.

### Command run

```
docker run --rm -it --network host --volume "$PWD:/app" dannyben/snapcrawl https://gridfinity.xyz/specification/ depth=0 log_level=0 screenshot_delay=3 css_selector="#main_content"
```

### Expected outcome

Just have a screenshot of the main content rather than the entire page

**Comment by DannyBen** [comment]
Correct. Bug.

[I found the problem](https://github.com/DannyBen/snapcrawl/pull/49/files#diff-4de6dade14e4cf5ba284e56c94905f8410927f548c6121d4e5d85a9d00383825) - but due to the fact that snapcrawl relies on aging dependencies, it gets harder to maintain. Tests are failing, and I am not sure I have the energy for it.

**Comment by DannyBen** [comment]
You would have to change the line so that:

1. You `git clone` the repository and `cd` into it.
2. Run `bundle` to install Ruby dependencies.
3. Run `bundle exec run gem build --install` to build and install the gem.

**Comment by DannyBen** [comment]
I will either try harder to fix the failing tests, or archive the repository. The aging dependencies have no replacement.

**Comment by DannyBen** [comment]
Good news. Tests are passing. I will release a pre-release in a few, and if it works for you, I will release a version.

**Comment by wburningham** [comment]
Awesome! Thanks for the fast turn-around time.

**Comment by DannyBen** [comment]
> Awesome! Thanks for the fast turn-around time.

I either do it quickly or not at all :)

Version 0.5.4.rc1 is [on rubygems](https://rubygems.org/gems/snapcrawl/versions/0.5.4.rc1)
Docker will take a few more minutes

**Comment by DannyBen** [comment]
Docker is also updated.
If it works for you - I will release 0.5.4.

**Comment by wburningham** [comment]
I just tested `dannyben/snapcrawl:0.5.4.rc1` locally and it correctly captures using the `css_selector`

**Comment by DannyBen** [comment]
Alright. I will let it soak for the night and release a version tomorrow.

**Comment by DannyBen** [comment]
Fixed in version 0.5.4.
